tests/unit/rules/css/test.invalidNesting.js

```diff
@@ -22,6 +26,23 @@ describe('CSS Rule InvalidNesting', () => {
     expect(linterMessages[0].type).toEqual(VALIDATION_WARNING);
   });
 
+  it('should not report invalid nesting on sufficient version', async () => {
+    const code = oneLine`/* I'm a comment */
+      #something {
+        .bar {
+          height: 100px;
+        }
+      }`;
+    const cssScanner = new CSSScanner(code, 'fakeFile.css', {
+      addonMetadata: {
+        firefoxMinVersion: '117.0.1',
```

Comment:
It'd be good to test with `117` too since you introduced a different comparison function to allow equal values.

Reply:
I added a test